### PR TITLE
odim:5522: Manager resource to be updated as per comments from OCP profile validation

### PR DESCRIPTION
**Pull request by lpooja**
default parameters  for odimra and plugin managers

### CODE REVIEW

**Review by irappa-pattar — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by hemkapoor — APPROVED** [review]

File: svc-managers/main.go [hunk 1]
```diff
@@ -93,8 +93,8 @@ func addManagertoDB(managerInterface mgrcommon.DBInterface) error {
 		ID:              config.Data.RootServiceUUID,
 		UUID:            config.Data.RootServiceUUID,
 		State:           "Enabled",
+		Health:          "OK",
```
Comment (amar-shalgar) [inline svc-managers/main.go:96]: Is this redfish property?

File: svc-managers/managers/managers.go [hunk 2]
```diff
@@ -191,8 +193,12 @@ func (e *ExternalInterface) getManagerDetails(id string) (mgrmodel.Manager, erro
 		Status: &mgrmodel.Status{
 			State: mgrData.State,
 		},
-		Description: mgrData.Description,
-		LogServices: mgrData.LogServices,
+		Description:         mgrData.Description,
+		LogServices:         mgrData.LogServices,
+		Model:               "ODIMRA" + " " + mgrData.FirmwareVersion,
```
Comment (amar-shalgar) [inline svc-managers/managers/managers.go]: model and powerstate can be saved in DB

File: svc-managers/managers/managers.go [hunk 3]
```diff
@@ -440,6 +445,15 @@ func fillResponse(body []byte, managerData map[string]interface{}) response.RPC
 		return common.GeneralError(http.StatusInternalServerError, response.InternalError, err.Error(),
 			[]interface{}{}, nil)
 	}
+	if _, ok := respData["DateTime"]; !ok {
```
Comment (amar-shalgar) [inline svc-managers/managers/managers.go]: This should not be checked as manager for odimra and plugin should always populate current datetime

File: svc-managers/mgrmodel/manager.go [hunk 4]
```diff
@@ -123,6 +123,7 @@ type RAManager struct {
 	State           string     `json:"State"`
 	Description     string     `json:"Description"`
 	LogServices     *dmtf.Link `json:"LogServices"`
+	Health          string     `json:"Health"`
```
Comment (amar-shalgar) [inline svc-managers/mgrmodel/manager.go:126]: is health a redfish property?

**Review by amar-shalgar — CHANGES_REQUESTED** [review]
Please address comments

File: svc-managers/managers/managers.go (hunk 3 above)
Comment (lpooja) [reply svc-managers/managers/managers.go]: addressed

File: svc-managers/managers/managers.go (hunk 2 above)
Comment (lpooja) [reply svc-managers/managers/managers.go]: addressed

File: svc-managers/mgrmodel/manager.go (hunk 4 above)
Comment (lpooja) [reply svc-managers/mgrmodel/manager.go:126]: this struct is for saving odimra details to DB, hence added it here

File: svc-managers/main.go (hunk 1 above)
Comment (lpooja) [reply svc-managers/main.go:96]: Yes

**Review by amar-shalgar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**